Tools/machines/lumi-csc/submit.sh

```diff
@@ -27,4 +25,19 @@ export FI_MR_CACHE_MONITOR=memhooks  # alternative cache monitor
 export ROCFFT_RTC_CACHE_PATH=/dev/null
 
 export OMP_NUM_THREADS=1
-srun ../warpx inputs > outputs
+
+cat << EOF > select_gpu
```

Comment:
Please add a comment as we do for other options (see above) :)

Reply:
I've added a few comments

Reply:
essentially I do what LUMI documentation recommends, so I put some references to the LUMI documentation